faqs/galaxy/datasets_datasets_not_downloading_at_all.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Dataset(s) not downloading at all?
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+1. Check to see if pop-ups are blocked by your web browser. Where to check can vary by browser and extensions.
+2. Double check your API key, if used. User > Preferences > Manage API key
```

Comment:
This again refers to downloading via the command line? maybe needs to be split up? @jennaj?